common/lib/xmodule/xmodule/templates/problem/problem_with_hint.yaml

```diff
@@ -4,13 +4,12 @@ metadata:
     markdown: !!null
 data: |
     <problem>
-      <text>
-        <p>
-          <h4>Problem With Adaptive Hint</h4>
-        </p>
-        <p>
-    This problem demonstrates a question with hints, based on using the <tt class="tt">hintfn</tt> method. </p>
-        <script type="text/python" system_path="python_lib">
+        <text>
+            <p><h4>Problem With Adaptive Hint</h4></p>
```

Comment:
I know you're bringing the same tagging forward, just changing the line feeds and indentation, but is it really necessary to have both the < p > and < h4 > tags for this line?